### PR TITLE
MEN-4208: Remove device on decommissioning

### PR DESCRIPTION
**Pull request by tranchitella**

### CODE REVIEW

**Comment by tranchitella** [comment]
Manual pipeline: https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/230507210
Depends on: https://github.com/mendersoftware/deviceauth/pull/442

**Comment by tranchitella** [comment]
PIpeline is green, but not reported because I skipped the Client acceptance tests to limit costs.

File: worker/provision_device.json [hunk 1]
```diff
@@ -1,48 +1,49 @@
...
+            }
+        }
+    ],
+    "inputParameters": [
+        "request_id",
+        "authorization",
+        "device_id",
+        "tenant_id"
+    ]
+}
```
Comment (alfrunes) [inline worker/provision_device.json:49]: > No newline at end of file

**Review by alfrunes — APPROVED** [review]
LGTM

Congrats with the #100 :)

File: worker/provision_device.json (hunk 1 above)
Comment (tranchitella) [reply worker/provision_device.json]: Added

**Pull request merged**